.changeset/smooth-ducks-sell.md

```diff
@@ -0,0 +1,9 @@
+---
+"@latticexyz/common": minor
```

Comment:
so according to semver, this should be `minor` because its an extra feature, not a fix

but maybe we don't want to trigger a bump to 2.1 yet?